src/ray/raylet/scheduling/policy/bundle_scheduling_policy.cc

```diff
@@ -0,0 +1,411 @@
+// Copyright 2021 The Ray Authors.
```

Comment:
seems some of the code is just moving old code, while the others involves refactor. can you mark those code actually changed and need review in this file?